pom.xml

```
@@ -17,9 +17,9 @@
     <url>https://wiki.jenkins.io/display/JENKINS/Localization+Support+Plugin</url>
```

```suggestion
    <url>https://github.com/jenkinsci/localization-support-plugin</url>
```

If you're up for a somewhat larger contribution: This plugin currently has two independent documentation pages. https://plugins.jenkins.io/localization-support/ shows the (read-only, deprecated) wiki content with a short introduction and changelog, while the GitHub repo has the developer focused docs.

Referencing the GH repo here would make https://plugins.jenkins.io/localization-support/ show the GH README; but of course it's lacking the changelog and is probably not a great first introduction for users.

So a slight adjustment of the GH README, adding a user-focused introduction, and migrating the changelog to a CHANGELOG.md and adding a link into the readme, would clean this up.